docs/appkit/unity/core/options.mdx

```diff
@@ -51,15 +51,11 @@ await AppKit.InitializeAsync(new AppKitConfig
             blockExplorer: new BlockExplorer("Seitrace", "https://seitrace.com"),
             rpcUrl: "https://evm-rpc-arctic-1.sei-apis.com",
             isTestnet: true,
-            imageUrl: $"https://api.web3modal.com/public/getAssetImage/692ed6ba-e569-459a-556a-776476829e00",
-            viemName: "seiDevnet" // required only on WebGL
+            imageUrl: $"https://api.web3modal.com/public/getAssetImage/692ed6ba-e569-459a-556a-776476829e00"
```

Comment:
This image is not public, can we instead use the CDN image URL from cloudflare? Like: 
 `https://imagedelivery.net/_aTEfDRm7z3tKgu9JhfeKA/692ed6ba-e569-459a-556a-776476829e00/md`